### PR TITLE
Upgrades Who verb, makes it as separate window

### PR DESCRIPTION
**Pull request by quardbreak**
Code is ugly, but now in separate window.
image

### CODE REVIEW

File: code/game/verbs/who.dm [hunk 1]
```diff
@@ -1,63 +1,94 @@
-
 /client/verb/who()
 	set name = "Who"
 	set category = "OOC"
 
-	var/msg = "<b>Current Players:</b>\n"
+	var/msg = "<meta charset='UTF-8'><h2><b>Curret Players:</b></h2>\n"
```
Comment (Lohikar) [inline code/game/verbs/who.dm]: Current.

File: code/game/verbs/who.dm [hunk 2]
```diff
@@ -1,63 +1,94 @@
...
 
+	var/living = 0
+	var/dead = 0
+	var/observers = 0
+	var/lobby = 0
+	var/living_antags = 0
+	var/dead_antags = 0
+
 	if(check_rights(R_INVESTIGATE, 0))
+		log_admin("[key_name(usr)] checked advanced who in-round.")
```
Comment (Lohikar) [inline code/game/verbs/who.dm:17]: Unlogged change.

Comment (MistakeNot4892) [reply code/game/verbs/who.dm:17]: Also seems kinda unnecessary, why does the admin team need to know when you use who?

File: code/game/verbs/who.dm [hunk 3]
```diff
@@ -1,63 +1,93 @@
...
+
+	msg += "<b>Total Players: [length(Lines)]</b><br><br>"
 
 	for(var/line in sortList(Lines))
-		msg += "[line]\n"
+		msg += "[line]<br>"
 
-	msg += "<b>Total Players: [length(Lines)]</b>"
-	to_chat(src, msg)
+	show_browser(usr, msg, "window=who;size=780x420;can_close=1")
```
Comment (MistakeNot4892) [inline code/game/verbs/who.dm:90]: Please use `/datum/browser` for adding new static windows like this.

**Comment by MistakeNot4892** [comment]
A preference for browser vs. chat window would probably be preferrable.
If it's being dumped to a window I think it could also benefit from a table to make the various fields align properly.

File: code/game/verbs/who.dm (hunk 2 above)
Comment (Lohikar) [reply code/game/verbs/who.dm:17]: It looks like it's specifically for admin-who, which does show stuff like antag status.

**Pull request closed without merging**